src/StdChains.sol

```diff
@@ -150,32 +164,36 @@ abstract contract StdChains {
         initialized = true;
 
         // If adding an RPC here, make sure to test the default RPC URL in `testRpcs`
-        setChainWithDefaultRpcUrl("anvil", Chain("Anvil", 31337, "http://127.0.0.1:8545"));
+        setChainWithDefaultRpcUrl("anvil", Chain("Anvil", 31337, "", "http://127.0.0.1:8545"));
```

Comment:
Did anything change in this section aside from adding the `""` throughout for chain alias? I don't think so but the diff came out annoying so just checking 😅

Reply:
Also this is a breaking change to the interface which I'd like to avoid if we can. What do you think of creating a new struct called `ChainData` and using that as the chain return data and keeping `Chain` as the input data?

Reply:
> Did anything change in this section aside from adding the `""` throughout for chain alias? I don't think so but the diff came out annoying so just checking 😅

Yes this is just the `""` addition.

> Also this is a breaking change to the interface which I'd like to avoid if we can. What do you think of creating a new struct called `ChainData` and using that as the chain return data and keeping `Chain` as the input data?

Changing the output is also a breaking change. Does it make more sense to break the input or the output? If I had to guess breaking the output would actually be a larger breaking change as this is people defining custom chains vs using the common ones defined.

I can proceed if you think this is a better way to go, but wanted to double check.